src/e2e/java/uk/gov/hmcts/reform/sscs/functional/IssueFurtherEvidenceHandlerFunctionalTest.java

```diff
@@ -83,10 +84,10 @@ private void verifyEvidenceIssuedAndExistingReasonableAdjustmentRaised() {
         SscsCaseData caseData = caseDetails.getData();
         List<SscsDocument> docs = caseData.getSscsDocument();
 
-        assertNull(docs.get(0).getValue().getEvidenceIssued());
-        assertEquals("Yes", docs.get(1).getValue().getEvidenceIssued());
-        assertEquals("Yes", docs.get(2).getValue().getEvidenceIssued());
-        assertEquals("Yes", docs.get(3).getValue().getEvidenceIssued());
+        Assertions.assertThat(docs)
+            .extracting(SscsDocument::getValue)
+            .extracting(SscsDocumentDetails::getEvidenceIssued)
+            .contains("Yes", "Yes", "Yes");
```

Comment:
.contains("Yes", "Yes", "Yes")
can be simplied/made more elegant by doing:
.hasSize(3)
.containsOnly("Yes")

Reply:
I'll make a call here and say that while Lucas' suggestion at first glance appears to be marginally better in this particular context, it's still subjective and I'm happy to keep the current implementation.

I'll mark this conversation as resolved.